components/org.wso2.carbon.business.rules.core/src/main/java/org/wso2/carbon/business/rules/core/services/TemplateManagerService.java

```diff
@@ -634,6 +624,13 @@ public void deployTemplates(String nodeURL, BusinessRuleFromTemplate businessRul
 
     public Map<String, BusinessRule> loadBusinessRules() {
         QueryExecutor queryExecutor = new QueryExecutor();
+        try {
+            queryExecutor.createTable();
+        } catch (SQLException e) {
+            log.error("A table for storing business rules in the database is not existed and cannot be created.");
```

Comment:
append the exception (`e`) to the log as well.

Reply:
Fixed by a5dad15bfe3f519314032f37e268f8363b2c64e4